src/main/java/Duke.java

```diff
@@ -1,10 +1,100 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String DOTTED_LINE = "____________________________________\n";
+    private static int numOfTasks = -1;
+    private static final int MAX_TASKS = 100;
+
+    private static Task[] tasks = new Task[MAX_TASKS];
+
+    private static void doneCommand(String input) {
+        int pos = Integer.parseInt(input.split(" ")[1]) - 1;
```

Comment:
Coding violation: Is there a better name for this than pos? What does pos refer to?